modules/acl/src/main/java/org/dcache/acl/mapper/AclMapper.java

```diff
@@ -78,10 +75,9 @@ public static Permission getPermission(Subject subject, Origin origin, Owner own
         } catch (ACLException e) {
             logger.error(e.getMessage());
         } finally {
-            if ( logger.isDebugEnabled() ) {
```

Comment:
I guess, the is statement was there to prevent 'expensive' **permACL.asString(rsType)**